backend/helpers.py

```diff
@@ -0,0 +1,14 @@
+
+from jina import DocumentArray, Document
+
+'''
+Creates a DocumentArray from a chosen column of a csv file.
+'''
```

Comment:
I think the docstring is supposed to be inside the function, as per https://www.python.org/dev/peps/pep-0257/

Reply:
Good catch!